sdks/python/apache_beam/runners/portability/flink_uber_jar_job_server.py

```diff
@@ -68,7 +68,7 @@ def executable_jar(self):
               'the scheme is specified. If using a local file path, make sure '
               'the file exists; you may have to first build the job server '
               'using `./gradlew runners:flink:%s:job-server:shadowJar`.' %
-              (self._executable_jar, self._flink_version))
+              (self._executable_jar, self.flink_version())
```

Comment:
Is the server always up when we make this call?

Reply:
Fixed.

Reply:
SGTM